core/src/sumeragi/mod.rs

```diff
@@ -775,13 +771,8 @@ impl<G: GenesisNetworkTrait, K: KuraTrait, W: WorldTrait> Sumeragi<G, K, W> {
                     .expect("Failed to sign CommitTimeout")
                     .into(),
                 ));
-                let mut send_futures = Vec::new();
-                for peer in &recipient_peers {
-                    if *peer != peer_id {
-                        send_futures.push(message.clone().send_to(&broker, peer));
-                    }
-                }
-                future::join_all(send_futures).await;
+
+                message.send_to_multiple(&broker, &recipient_peers).await;
```

Comment:
We have `msg` in half the places and `message` in the other half. Think we can make it consistent?

Reply:
I prefer abbreviations